mimic/model/nova_objects.py

```diff
@@ -527,6 +527,35 @@ def fail_later(server):
     return fail_later
 
 
+@server_creation.declare_behavior_creator("sequence")
+def sequence(parameters):
+    """
+    Sometimes a sequence of behaviors occur when you try to create a server in
+    a predictable pattern.
+
+    Takes one parameter, ``behaviors``, which is a list of specifications of
```

Comment:
Sorry, I mean they they are documented with name and parameters in a dictionary in the REST api, but not in the object model. The behaviors above describes the dictionary that is in parameters.

Whereas the behavior specification here is expecting a list of dictionaries as would be passed as the body to the REST api call.